api/v1/verticadb_types.go

```diff
@@ -840,6 +840,14 @@ type Proxy struct {
 	// use a secret path reference prefix, such as gsm://. Everything after the
 	// prefix is the name of the secret in the service you are storing.
 	TLSSecret string `json:"tlsSecret,omitempty"`
+}
+
+type ProxySubclusterConfig struct {
+	// +kubebuilder:default:=1
```

Comment:
Let's remove the default value otherwise the proxy field will always show up in the manifest even when client proxy is disabled.

Reply:
Done, updated in https://github.com/vertica/vertica-kubernetes/pull/985/commits/5135a6a307167492fc151fb108a08e94a7c9b0b0